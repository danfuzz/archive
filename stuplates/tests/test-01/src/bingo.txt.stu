"I am the bingo script.\n";
