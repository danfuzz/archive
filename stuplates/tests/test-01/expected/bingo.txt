I am the bingo script.
